package.json

```diff
@@ -241,7 +241,6 @@
     "@expo/config-plugins/glob": "^10.3.10",
     "@solana/web3.js/rpc-websockets": "^8.0.1",
     "@metamask/message-manager": "^10.1.0",
-    "@metamask/gas-fee-controller@npm:^15.1.1": "patch:@metamask/gas-fee-controller@npm%3A15.1.2#~/.yarn/patches/@metamask-gas-fee-controller-npm-15.1.2-db4d2976aa.patch",
```

Comment:
This resolution was already unused. Not directly related to the bump here, but couldn't hurt to remove it.